test/index.spec.js

```diff
@@ -248,24 +248,20 @@ describe('discipl-ephemeral-connector', () => {
     it('should remove stale identities', async () => {
       let ephemeralConnector = new EphemeralConnector()
       ephemeralConnector.configure(EPHEMERAL_ENDPOINT, EPHEMERAL_WEBSOCKET_ENDPOINT, w3cwebsocket)
-
       let identity = await ephemeralConnector.newIdentity()
-
       let claimLink = await ephemeralConnector.claim(identity.did, identity.privkey, { 'THIS': 'WILL_DISAPPEAR' })
-
       let claim = await ephemeralConnector.get(claimLink, identity.did, identity.privkey)
-
       expect(claim).to.deep.equal({
         'data': {
           'THIS': 'WILL_DISAPPEAR'
         },
         'previous': null
       })
-
       await timeoutPromise(1500)
 
-      let expiredClaim = await ephemeralConnector.get(claimLink, identity.did, identity.privkey)
+      ephemeralConnector.deleteAllFromCache()
```

Comment:
I would actually repeat the get/test that there is data after the timeout but before deleteAllFromCache, since that way we test that the client-side cache works. I would also add some comments specifying what we're testing with each expect.